src/components/charting/drawChart.jsx

```diff
@@ -1,12 +1,24 @@
 import { computed } from "@preact/signals-react";
 import React from "react";
+import { indicatorConfig } from "../../config/indicatorsConfig";
 
-function DrawChart({
-  handleOnMouseMove,
-  removeCursor,
-  drawChart,
-}) {
-  const indicatorsLength = computed(() => drawChart.ChartWindow.offChartIndicatorSignal.value.length);
+function DrawChart({ handleOnMouseMove, removeCursor, drawChart, mode }) {
+  setTimeout(() => {
+    drawChart.drawChartFunction(drawChart, mode);
+    if (localStorage.getItem("offChartIndicators") !== null ) {
+      const indicators = [];
+      const {offChartIndicatorSignal} = drawChart.ChartWindow;
+      const storedIndicators = JSON.parse(localStorage.getItem("offChartIndicators"));
+      if(offChartIndicatorSignal.peek().length === storedIndicators.length)return;
+      storedIndicators.forEach((indicator)=>{
+        indicators.push(indicatorConfig[indicator.label]);
+      })
+      drawChart.ChartWindow.offChartIndicatorSignal.value = [...indicators];
```

Comment:
> React has detected a change in the order of Hooks called by Charting. This will lead to bugs and errors if not fixed. For more information, read the Rules of Hooks: https://reactjs.org/link/rules-of-hooks

Facing this issue while trying to run `RSI indicator` for the first time, i.e., when we first set the `offChartIndicatorSignals` in local storage for the first time. Please check this out

Reply:
Rebase the PR with main